libs/dh/wholesale/feature-calculations/src/lib/details/details.component.scss

```diff
@@ -28,6 +28,11 @@ watt-description-list {
   margin-left: var(--watt-space-ml);
 }
 
+watt-drawer-content {
+  display: grid;
+  grid-template-rows: auto minmax(24rem, 1fr);
+}
+
```

Comment:
question: what's the purpose of this? Doesn't it currently work somewhat similarly?

Reply:
Yeah, I started some work on the details drawer, but then later decided to do it later to keep the PR small 😄 So it sort of does nothing right now, but I'll need it later. Do you want me to remove it to keep the PR clean? 🙂

Reply:
Don't have to remove it. 

Well, it has a small side effect where that the `margin` on the `dl` elements inside `watt-description-list` in the drawer is now taken into account, whereas before the two `margin`s collapsed. That's why I was wondering about its purpose 😊

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/d49d64b9-b4ce-41b2-80d3-cee73fbd1394)

Reply:
Good catch! I have removed the change, that was not intended! Thanks for the help!